### DISCUSSION TITLE
navigating back by nuxt-link adds a trailling slash automatically

### DISCUSSION THREAD

**Discussion by polarove** [nuxt/nuxt#29625] [Questions]
```vue
<nuxt-link to="..">Back</nuxt-link>
```
This would navigate me back to `http://example.com/path/` instead of `http://example.com/path`

Is it by design or?

**Comment by xjccc** [comment]
[nuxt-link](https://nuxt.com/docs/api/components/nuxt-link#nuxtlink)

You could see `trailingSlash` in `nuxt-link`. And check your version or config

**Reply by polarove** [reply]
Thanks!
